apisix/plugins/response-rewrite.lua

```diff
@@ -33,6 +33,11 @@ local schema = {
             description = "new body for repsonse",
             type = "string",
         },
+        body_base64 = {
```

Comment:
we can check if it is a valid `base64` value in `function _M.check_schema(conf)`.

Reply:
done